subsys/bluetooth/controller/hci_driver.c

```diff
@@ -205,7 +231,7 @@ static void signal_thread(void *p1, void *p2, void *p3)
 
 	while (true) {
 		k_sem_take(&sem_signal, K_FOREVER);
-		blectlr_signal();
+		ble_controller_low_prio_tasks_process();
```

Comment:
It seems to me that this function should also do MULTITHREADING_LOCK_ACQUIRE/RELEASE